src/components/AppNavigation/CircleNavigationItem.vue

```diff
@@ -35,16 +35,28 @@
 			<ActionButton
 				v-if="circle.canManageMembers"
 				:close-after-click="true"
-				icon="icon-add"
 				@click="addMemberToCircle">
+				<template #icon>
+					<IconAdd
+						:size="20" />
+				</template>
 				{{ t('contacts', 'Add member') }}
 			</ActionButton>
 
 			<!-- copy circle link -->
 			<ActionLink
 				:href="circleUrl"
-				:icon="copyLinkIcon"
 				@click.stop.prevent="copyToClipboard(circleUrl)">
+				<!-- TODO Change the IconLoading into the proper loading one once new vue is released, icon-loading -->
```

Comment:
https://github.com/nextcloud/nextcloud-vue/pull/2727 right?

Reply:
but its not released

Reply:
alright, then let's link this from the PR description then

Reply:
see descr